ibis/backends/base/__init__.py

```diff
@@ -83,7 +83,7 @@ def current_database(self) -> str | None:
         """
 
     @abc.abstractmethod
-    def list_databases(self, like: str = None) -> List[str]:
+    def list_databases(self, like: str = None) -> list[str]:
```

Comment:
i think this typing sytntax is fine, but requires `from __future__ import annotations` in every module that supports it in order for this to not break 3.7. odd that the CI is passing.

Reply:
`from __future__ import annotations` is already in this module. If you expand the file in the GitHub UI you can see it on line 1.